apps/prairielearn/src/pages/administratorQuery/administratorQuery.html.ts

```diff
@@ -234,13 +129,13 @@ export function AdministratorQuery({
             ${query_run
               ? html`
                   <div class="card-body d-flex align-items-center p-2 bg-secondary text-white">
-                    Query ran at: ${query_run?.formatted_date}
+                    Query ran at: ${query_run.date ? formatDate(query_run.date, 'UTC') : 'unknown'}
                     ${query_run?.result != null
                       ? html`
                           <div class="ml-auto">
-                            <span class="mr-2 test-suite-row-count">
-                              ${query_run.result.rowCount}
-                              ${query_run.result.rowCount === 1 ? 'row' : 'rows'}
+                            <span class="mr-2" data-test-id="row-count">
```

Comment:
The attribute should be `data-testid`, not `data-test-id`. Ditto for other occurrences of this attribute.

Reply:
Done in aefb4613e8b06dcd662a1c9a00084b6b37236d10.